pkg/controller/core.oam.dev/v1alpha2/application/testcmd/server.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"time"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+
+	core "github.com/oam-dev/kubevela/pkg/controller/core.oam.dev"
+	"github.com/oam-dev/kubevela/pkg/controller/core.oam.dev/v1alpha2/application"
+)
+
+var (
+	setupLog = ctrl.Log.WithName("setup")
+)
+
+func newServerCommand() *cobra.Command {
+	options := new(serverOptions)
+	cmd := &cobra.Command{
+		Use:  "vela-server",
```

Comment:
```suggestion
		Use:  "vela server",
```

Reply:
we don't need this command even only for test